facade/logstash_test.go

```diff
@@ -172,6 +267,46 @@ func TestConstructingFilterString(t *testing.T) {
 	}
 }
 
+func TestGetAuditLogSectionForNoservice(t *testing.T) {
+	services := []servicedefinition.ServiceDefinition{}
+	auditLogTypes := []string{}
+	auditLogSection := getAuditLogSection(services, &auditLogTypes)
+	if len(auditLogSection) > 0 {
+		t.Error(fmt.Sprintf("Audit Log Section should be empty but it is not %d", len(auditLogSection)))
```

Comment:
if this test fails in the future, it would speed up debugging if the error message also printed the value of `auditLogSection`